### DISCUSSION TITLE
Windows Terminal 1.19.10573.0 cannot launch any tabs -- error 259

### DISCUSSION THREAD

**Discussion by skycommand** [microsoft/terminal#16796] [Q&A] [answered]
Just an hour ago, I updated Windows Terminal on one of my PCs to version 1.19.10573.0. Unfortunately, I cannot use it anymore.

Windows Terminal disappears a few seconds after the launch unless I press Ctrl+Comma to open the Settings tab. After that, attempting to launch any tab of any kind (PowerShell, Legacy PowerShell, Command Prompt, or Azure Console) fails. The tab disappears after a second.

I used a screen capture tool to record the tab. Through frame-by-frame viewing of the video, I noticed that the shell process throws an error 259 before quitting.

I tried switching off the Atlas rendering engine, but it had no effect.

Any ideas as to what is wrong?

**Comment by DHowett** [comment, marked as answer]
Let me guess... you've got an Intel Core 2 Duo or similar?

If so, this may be #16794

**Reply by skycommand** [reply]
Yes. I've repurposed that system into a NAS because it's not good for anything else.
